src/planwise/client/projects2/handlers.cljs

```diff
@@ -69,9 +69,10 @@
 ;; upon callback we are updating the local db current-project
 (rf/reg-event-fx
  :projects2/start-project
- in-projects2
+ ;in-projects2
  (fn [{:keys [db]} [_ id]]
-   {:api (assoc (api/start-project! id)
+   {:db (assoc-in db [:scenarios :list-scope] nil)
```

Comment:
You should probably `invalidate` the `asdf` which contains the list of scenarios instead. Ditching `list-scope` altogether would be even better.